src/collections/config/unit.test.ts

```diff
@@ -122,8 +124,22 @@ describe('Unit testing of the MergeWithExisting class', () => {
     autoTenantCreation: false,
   };
 
+  const moduleConfig: WeaviateModuleConfig = {
+    'generative-cohere': {
+      kProperty: 0.1,
+      model: 'model',
+      maxTokensProperty: '5',
+      returnLikelihoodsProperty: 'likelihoods',
+      stopSequencesProperty: ['and'],
+      temperatureProperty: 5.2,
+    },
+    'reranker-cohere': {},
+  };
+
+  const parse = (config: any) => JSON.parse(JSON.stringify(config));
```

Comment:
Wouldn't this be a no-op most of the time?

Reply:
Yes and no. It won't change the values but it acts as a copy of the config so that the defined configs can be reused between tests. This is because the `MergeWithExisting` static methods modify the passed `current` configs due to copy by reference